applications/StructuralMechanicsApplication/tests/test_patch_test_cr_beam.py

```diff
@@ -155,7 +174,20 @@ def _solve_dynamic(self,mp):
         
         strategy.Check()
         strategy.Solve()
-        
+
+    def _set_dynamic_explicit_strategy(self,mp):
```

Comment:
name is misleading imo
I would do sth like `_create_dynamic_explicit_strategy`

and then later call `Solve()` directly